Cargo.toml

```diff
@@ -101,8 +101,8 @@ members = [
     "tesseract/consensus/sync-committee",
     "tesseract/consensus/grandpa",
     "tesseract/consensus/integration-tests",
-    "tesseract/consensus/beefy",
-    "tesseract/consensus/beefy/zk",
+#    "tesseract/consensus/beefy",
+#    "tesseract/consensus/beefy/zk",
```

Comment:
@seunlanlege @dharjeezy Yes, sorry - this was temporary, just to pass test compilation without your private repo. I shouldn’t have pushed it. Also, I didn’t expect it to be merged - more like just to show the diff with the fix, but no problem :)